subprojects/ce-login/uploadDevAcf

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Script to generate a development signed ACF and upload to the BMC provided
+
+
+MYPATH=$(dirname $(readlink -f $(which $0)));
+BMC="$1"
+ARCH="$2"
+PASSWD="$3"
+ADMINID=admin
```

Comment:
I was able to remove the hack to use root as with the 31.12 BMC release I don't hit the forbidden error anymore.  I was also able to test the entire script and login with the service account after doing the install